_rules/required-form-fields-not-completed-334972.md

```diff
@@ -0,0 +1,243 @@
+---
+id: 334972
+name: Error message identifies required fields left unfilled
+rule_type: atomic
+description: |
+  This rule checks that text error messages identify required form fields that were not completed.
+accessibility_requirements:
+  wcag20:3.3.1: # Error Identification (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G83: # Providing text descriptions to identify required fields that were not completed
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree # (e.g. HTTP Messages, DOM Tree, CSS Styling, Accessibility Tree, Language, etc.,)
+  - CSS Styling
+  - Language
+acknowledgements:
+  authors:
+    - Carlos Duarte
+    - João Vicente
+---
+
+## Applicability
+
+The rule applies to each [required][] HTML element that:
+
+- is a [descendant][] of a [form element][]; and
+- has one of the following [semantic roles][semantic role]: `checkbox`, `combobox`, `listbox`, `menuitemcheckbox`, `menuitemradio`, `radio`, `searchbox`, `slider`, `spinbutton`, `switch` and `textbox`.
+
+**Note**: The list of applicable [semantic roles][semantic role] is derived by taking all the [ARIA 1.1][] roles that:
+
+- inherit from the [abstract][] `input` or `select` role, and
+- do not have a [required context][] role that itself inherits from one of those roles.
+
+## Expectation 1
+
+After triggering the submission of the [form element][] that is an [ancestor][] of the target element, each target element that was not [interacted with][] is identified by a text message.
```

Comment:
Same comment as for the previous rule. I think getting a form field into an error state should not be part of this rule. Just say what the requirements are for when it is in an error state.